app/src/main/java/com/example/sabonit/ProductActivity.java

```diff
@@ -89,6 +93,44 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         literSeekBar.setProgress(initalProgress);
     }
 
+    private void fillBottle(double numLiters, int color) {
+        final ContextThemeWrapper wrapper = new ContextThemeWrapper(this, color);
```

Comment:
the name of the variable "numLiters" is not informative, is it the numLiters of the bottle or numLiters that the user chose?

Reply:
OK you are right we will change it.
it means the numLiters the seek bar shows right now